lib/backend.js

```diff
@@ -60,7 +60,6 @@ exports.createServer = (dynamodb, docClient) => {
     docClient = docClient || new AWS.DynamoDB.DocumentClient({service: dynamodb})
   }
 
-  const listTables = promisify(dynamodb.listTables.bind(dynamodb))
```

Comment:
What's the reason for not using promisified version? Should result in smaller, cleaner and more readable code.

Reply:
I thought about it more. I guess it wouldn't look that clean unless using async/await and maybe we should not introduce those keywords for compatibility reasons.

So keep it that way for now. :)

Reply:
Actually...

```
  app.get('/', (req, res) => {
    const listAllTables = (lastEvaluatedTableName, tableNames) => {
      return listTables({ ExclusiveStartTableName: lastEvaluatedTableName })
        .then(data => {
          tableNames = tableNames.concat(data.TableNames)

          if (typeof data.LastEvaluatedTableName !== 'undefined') {
            return listAllTables(data.LastEvaluatedTableName, tableNames);
          }

          return Promise.all(
            tableNames.map(
              TableName => describeTable({ TableName }).then(data => data.Table)));
        });
    }

    listAllTables(null, [])
      .then(data => {
        res.render('tables', { data })
      })
      .catch(error => {
        res.json({ error })
      })
  })
```

seems nicer.

Reply:
agreed, changed